front-end/src/domains/JobInfoItem/JobInfoItem.tsx

```diff
@@ -39,21 +39,23 @@ const JobInfoItem = ({
   memberName,
   refresh,
 }: JobInfoItemProps) => {
-  // TODO: 실제 예상 시간으로 교체
-  // TODO: 서버별 job의 현재 실행 중인 job의 예상 종료 시간 + 나머지 jobs들의 expected time을 전부 더함
-
   const history = useHistory();
 
   const { myInfo } = useAuth() as { myInfo: MyInfoResponse };
```

Comment:
```suggestion
  const labId = useLabId();
```

또는 page에서 `uselabId`를 호출해서 `labId`를 가져온 다음에, prop drilling으로 `labId`를 가지고 와도 좋을거 같아요. 
`labId`가 `null`인 경우는 정상적인 상황이 아니므로, page초기 진입시에 error를 터뜨리는게 좋지 않을까요

Reply:
a2ca2da

나머지 컴포넌트에서 `labId=1` 형태로 fix했기 때문에 **우선** 동일한 방식으로 변경했습니다.
이 부분은 아마 동동이 `useAuth`관련 해서 해결해줬기 때문에 **추후에 `useLabId`를 사용해도 괜찮을 것 같습니다.**

Reply:
일부 페이지는  `labId`를 내려주고 있을거에요
한번에 다 처리하기 귀찮아서, 눈에 보일 때마다 처리하고 있습니다
다음에는 일관성 생각하지 말고 그냥 눈에 보이는건 바로 리팩토링 해주세요 :+1: